install/empty_data.php

```diff
@@ -2965,6 +2979,11 @@
       'notifications_id'         =>  '67',
       'mode'                     =>  'mailing',
       'notificationtemplates_id' =>  25,
+   ], [
+      'id'                       => 68,
+      'notifications_id'         =>  '68',
+      'mode'                     =>  'mailing',
+      'notificationtemplates_id' =>  26,
    ],
```

Comment:
You should also add an entry for notification `69` ('Alert domains close expiries').